core/templates/dev/head/css/oppia.css

```diff
@@ -1026,8 +1089,8 @@ md-card.oppia-gallery-tile {
 }
 
 .oppia-html-select .oppia-html-select-option oppia-noninteractive-image {
```

Comment:
There's a whole bunch of stuff here that seems to have been deleted/changed, but shouldn't be. Please could you reinstate it?